client/src/main/fbthrift/com/facebook/thrift/transport/OkHttp3Util.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2023 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+package com.facebook.thrift.transport;
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import okhttp3.OkHttpClient;
+import okhttp3.Protocol;
+
+public class OkHttp3Util {
+    private static OkHttpClient client;
+
+    private OkHttp3Util() {
+    }
+
+    public static OkHttpClient getClient(int connectTimeout, int readTimeout,
+                                         SSLSocketFactory sslFactory,
+                                         TrustManager trustManager) {
+        if (client == null) {
+            synchronized (OkHttp3Util.class) {
+                if (client == null) {
+                    // Create OkHttpClient builder
+                    OkHttpClient.Builder clientBuilder = new OkHttpClient.Builder()
+                            .connectTimeout(connectTimeout, TimeUnit.MILLISECONDS)
+                            .writeTimeout(readTimeout, TimeUnit.MILLISECONDS)
```

Comment:
is writeTimeout have same value with readTimeOut?

Reply:
yeah，we do use the same timeout value.